init.sls

```diff
@@ -1,3 +1,15 @@
+{% set ssl_protocols = salt['pillar.get']('nginx:ssl_protocols', 'TLSv1.1 TLSv1.2 TLSv1.3') %}
+{% set ssl_ciphers = salt['pillar.get']('nginx:ssl_ciphers', [
```

Comment:
I'd set this as a single string, makes copy & paste easier. Also can't be split up in the `pillar.example` below, so for comparing, etc. I think it would be a little nicer (even though it means long line)

Reply:
I changed it to be in a single line. 

I allow long lines also in salt-lint configuration.